src/ant/README.md

```diff
@@ -33,6 +33,18 @@ check_antennas [-net net] [-verbose]
 - `-verbose` report all antenna calculations for violating nets
 - 'net' check antennas on one net
 
+### `find_max_allowed_length`
+Find the maximum allowed length for a given net and layer
```

Comment:
this command name seems overly generic. It says nothing about the fact that it is related to antennas.